package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "wolox-equalizer",
+  "version": "1.0.0",
```

Comment:
I think version `0.0.1` would be better so a v1 is a really big and complete package :smile: